pkg/osx/GNUmakefile

```diff
@@ -60,26 +61,23 @@ $(STAGING_DIR): launcher $(TOPLEVEL_DOCS)
 	cp launcher "$(APP_BIN_DIR)"
 	$(STRIP) "$(APP_BIN_DIR)/launcher"
 
-	./cp-with-libs $(TOPLEVEL)/src/$(PROGRAM_PREFIX)doom "$(APP_BIN_DIR)"
-	$(STRIP) "$(APP_BIN_DIR)/$(PROGRAM_PREFIX)doom"
-	./cp-with-libs $(TOPLEVEL)/src/$(PROGRAM_PREFIX)heretic "$(APP_BIN_DIR)"
-	$(STRIP) "$(APP_BIN_DIR)/$(PROGRAM_PREFIX)heretic"
-	./cp-with-libs $(TOPLEVEL)/src/$(PROGRAM_PREFIX)hexen "$(APP_BIN_DIR)"
-	$(STRIP) "$(APP_BIN_DIR)/$(PROGRAM_PREFIX)hexen"
-	./cp-with-libs $(TOPLEVEL)/src/$(PROGRAM_PREFIX)strife "$(APP_BIN_DIR)"
-	$(STRIP) "$(APP_BIN_DIR)/$(PROGRAM_PREFIX)strife"
+	for game in $(GAMES); do	\
+		./cp-with-libs $(TOPLEVEL)/src/$(PROGRAM_PREFIX)$$game "$(APP_BIN_DIR)";	\
+		$(STRIP) "$(APP_BIN_DIR)/$(PROGRAM_PREFIX)$$game";	\
+	done
 	./cp-with-libs $(TOPLEVEL)/src/$(PROGRAM_PREFIX)setup "$(APP_BIN_DIR)"
 	$(STRIP) "$(APP_BIN_DIR)/$(PROGRAM_PREFIX)setup"
 
 	$(TOPLEVEL)/man/simplecpp -DPRECOMPILED -D__MACOSX__  \
 	                -DDOOM -DHERETIC -DHEXEN -DSTRIFE     \
+					-DLONG_GAME_NAME="Crispy Doom" \
```

Comment:
Why is this required for Crispy, but apparently not for Choco?

Reply:
Nice catch, apparently it's required for Chocolate too and it's also broken there (running the .dmg makefile for the current GitHub version of Chocolate Doom also produced an INSTALL file with "LONG_GAME_NAME" peppered throughout instead of "Chocolate Doom"). It looks like "Chocolate Doom" used to be hard-coded as LONG_GAME_NAME at the start of the INSTALL.template file, but they changed it to be specified in the makefile later without updating the .dmg makefile to match. I'll make a pull request there now that I've caught it.